supervisory-authorities/derfdw.json

```diff
@@ -0,0 +1,16 @@
+{
+    "slug": "derfdw",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Rundfunkdatenschutzbeauftragter Deutsche Welle",
+    "address": "Kurt-Schumacher-Straße 3\n53113 Bonn\nDeutschland",
+    "phone": "+49 228 4290",
```

Comment:
That seems to be the general phone number for DW (see: https://www.dw.com/de/service/kontakt/s-30583).

This page (https://www.rundfunkdatenschutzkonferenz.de/kontakt) lists more specific phone and fax numbers. I think we should use those.